webapps/docs/changelog.xml

```diff
@@ -143,6 +143,12 @@
         request line, ensure that all the available data is included in the
         error message. (markt)
       </fix>
+      <fix>
+        <bug>65272</bug>: Restore the optional HTTP feature that allows
+        <code>LF</code> to be treated as a line terminator as well as the
```

Comment:
This is specifically for a line-terminator for request-line and headers, correct? Is it worth mentioning that specifically?

Reply:
Can do.